tests/unit/test_forwarding_client.py

```diff
@@ -42,112 +57,100 @@ class ContentTypeTestCase(BaseTestCase):
     ],
     ids=str,
 )
-async def test_forward_request_with_basemodel_response(testcase: ContentTypeTestCase) -> None:
```

Comment:
Tests remain the same, `forward_client` is now a fixture and the context manager scope is removed.